lib/github_repositories.rb

```diff
@@ -65,8 +65,8 @@ def pass
         "default_branch_main": true,
         "has_default_branch_protection": true,
         "requires_approving_reviews": true,
-        "requires_code_owner_reviews": true,
         "administrators_require_review": true,
-#         "dismisses_stale_reviews": true,
-#         "team_is_admin": true,
-        "issues_enabled": true
+        "issues_section_enabled": true
+        "requires_code_owner_reviews": true,
```

Comment:
I think this one needs to go too

Reply:
I thought we were still doing the branch protection settings:
 1. Require a pull request before merging
    2. Require approvals option and a minimum number of user to approve the pull request.
    3. Require review from Code Owners option
    4. Include administrators option

Reply:
okay will need to do some more chopping in these two repo's